app/controllers/api/v2/bulk_snapshots_controller.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    class BulkSnapshotsController < V2::BaseController
+      api :POST, '/snapshots/bulk/create_snapshot', N_('Create snapshots for multiple hosts')
+      param :host_ids, Array, of: Integer, desc: N_('Array of host IDs to create snapshots for'), required: true
+
+      param :snapshot, Hash, desc: N_('Snapshot parameters'), required: true do
+        param :name, String, desc: N_('Name of the snapshot'), required: true
+        param :description, String, desc: N_('Description of the snapshot'), required: false
+      end
+
+      param :mode,
+        ['full', 'include_ram', 'quiesce'],
+        desc: N_('Snapshot mode: full, include_ram, or quiesce'),
```

Comment:
You could add to the description what is the default mode, as this is not a required parameter.